pkg/selinux/selinux.go

```diff
@@ -409,19 +409,21 @@ func GetLxcContexts() (processLabel string, fileLabel string) {
 	}
 
 	if processLabel == "" || fileLabel == "" {
-		return "", ""
+		return "", "", nil
```

Comment:
Luca raised the reasonable point put of band that we might want to move to
using the actual latest selinux package as a dependency so we don't have to
maintain our own (assuming of course that it's in a more sane state by now
- perhaps it isn't)

Matthew Garrett <notifications@github.com> schrieb am Di., 31. Mai 2016,
19:04:

> In pkg/selinux/selinux.go
> <https://github.com/coreos/rkt/pull/2723#discussion_r65222667>:
>
> > @@ -409,19 +409,21 @@ func GetLxcContexts() (processLabel string, fileLabel string) {
> >  	}
> >
> >  	if processLabel == "" || fileLabel == "" {
> > -		return "", ""
> > +		return "", "", nil
>
> Yeah I think that's reasonable. It's valid for selinux to be enabled and
> for us to have no lxc_contexts - there's no standard for selinux policy. In
> that case we should just continue without it.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/coreos/rkt/pull/2723/files/438c1e7197aedb6d83b10beff5cd6b8947a7c20a#r65222667>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACewNz07_v2Nar1pfbHgzynOsM8iL0aIks5qHGofgaJpZM4IqV-A>
> .
>